guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>I suggest we should first explain only the pooling module here. see <a href="https://towardsdatascience.com/review-pspnet-winner-in-ilsvrc-2016-semantic-segmentation-scene-parsing-e089e5df177d" target="_blank">https://towardsdatascience.com/review-pspnet-winner-in-ilsvrc-2016-semantic-segmentation-scene-parsing-e089e5df177d</a></p><p> </p><p>Like we have a separate section for  U-net implementation in&nbsp;<code>arcgis.learn</code>,  we should have similar for PSPNet implementation in arcgis.learn.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Since UnetClassifier and PSPNetClassifier have same methods to train, IMO there is no need to have seperate sample notebooks for each one. Therefore, I just specified the way to initialize PSPNet then the same workflow continues.</p>